source/products/hansemerkur/softfair/rest-api.rst

```diff
@@ -0,0 +1,524 @@
+REST API - Dog Surgery
+======================
+
+.. note::  Refer to :ref:`REST API page<rest_api>` for a more complete documentation regarding the necessary requests before performing and building your own.
+
+**Please remember to use the API specific item and touchpoint .**
+
+V2 Header
+----------
+
+The API requests must use the V2 header as show in the examples below:
+
+``Accept: application/vnd.kasko.v2+json``
+
+Quote Data
+^^^^^^^^^^
+Query string data appended to the quote request
+
+.. csv-table::
+   :header: "Name", "Type", "Description", "Example Value"
+   :widths: 20, 20, 80, 20
+
+   "gross_premium", "integer", "Gross premium calculated by the distributor", "10000000"
+   "policy_start_date", "string", "Policy start date in ISO date format", "2020-01-01"
+   "policy_end_date", "string", "Policy end_date in ISO date format", "2020-02-01"
+
+Example Request
+~~~~~~~~~~~~~~~
+
+.. code:: bash
+
+    curl -X GET \
+      'https://api.kasko.io/quotes' \
+      -H 'Accept: application/vnd.kasko.v2+json' \
+      -H 'Content-Type: application/json' \
+      -u <SECRET KEY>: \
+      -d '{
+        "item_id": "<ITEM ID>",
+        "touchpoint_id": "<TOUCHPOINT ID>",
+        "subscription_plan_id": "<SUBSCRIPTION PLAN ID>",
+        "data": {
+            "gross_premium": "100000",
+            "policy_start_date": "2020-01-01",
+            "policy_end_date": "2020-02-01"
+        }
+    }'
+
+Example Response
+~~~~~~~~~~~~~~~~
+
+.. code:: javascript
+
+    {
+        "token": "<QUOTE_TOKEN",
+        "gross_payment_amount": 100000,
+        "extra_data": {
+            "gross_premium": "100000",
+            "premium_tax": 15966,
+            "net_premium": 84034,
+            "tax_rate": 0.19
+        }
+    }
+
+Lead Request
+^^^^^^^^^^^^
+JSON data posted to /leads to create a lead
+
+.. csv-table::
+   :header: "Name", "Type", "Example Value"
+   :widths: 20, 20, 20
+
+   "touchpoint_id", "string", "<TOUCHPOINT_ID>"
+   "item_id", "string", "<ITEM_ID>"
+   "first_name", "string", "firstName"
+   "last_name", "string", "lastName"
+   "dob", "string", "1993-12-02"
+   "policy_start_date", "string", "2020-01-01"
+   "policy_end_date", "string", "2020-02-01"
+   "dog_name", "string", "dogName"
+   "dog_hybrid", "string", "dog hybrid"
+   "dog_breed", "string", "doberman"
+   "dog_dangerous", "string", "No"
+   "registration", "string", "Yes"
+   "coverage", "string", "Yes"
+   "deductible", "string", "Yes"
```

Comment:
Shouldn't all of these `no/yes` values be boolean - `true/false`?

Reply:
I also wondered this, but Vic said the client will send these values. How important do you think it is that we use booleans instead of strings there?

Reply:
it would seem more logical, but if the client will be sending that and it works then its ok

Reply:
makes sense! 👍